src/argus/notificationprofile/migrations/0010_notificationprofile_id.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.2.6 on 2022-03-11 15:24
+
+from django.db import migrations, models
+
+
+def copy_id_to_timeslot(apps, schema_editor):
+    NotificationProfile = apps.get_model("argus_notificationprofile", "NotificationProfile")
+
+    profiles = NotificationProfile.objects.all()
+    for profile in profiles:
+        profile.timeslot_id = profile.id
+
+    NotificationProfile.objects.bulk_update(objs=profiles, fields=["timeslot"])
+
+
+def fix_multi_use_timeslot_profiles(apps, schema_editor):
+    Timeslot = apps.get_model("argus_notificationprofile", "Timeslot")
+
+    timeslots = Timeslot.objects.prefetch_related("notification_profiles").annotate(
+        num_profiles=models.Count("notification_profiles")
+    )
+    for timeslot in timeslots.filter(num_profiles__gt=1):
+        i = 0
+        profiles = timeslot.notification_profiles.order_by("pk").all()
+        first_profile_id = profiles[0].id
+        for profile in profiles.exclude(id=first_profile_id):
+            new_timeslot = Timeslot.objects.get_or_create(user=timeslot.user, name="".join([timeslot.name, str(i)]))
+            i += 1
```

Comment:
Just a reminder that the Pythonic way of achieving this is normally thus:

```suggestion
        for i, profile in enumerate(profiles.exclude(id=first_profile_id)):
            new_timeslot = Timeslot.objects.get_or_create(user=timeslot.user, name="".join([timeslot.name, str(i)]))
```